packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useFetch, useInvalidateQuery } from '@deriv/api';
+import useExchangeRate from './useExchangeRate';
+
+/**
+ * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
+ */
+const useP2PAdvertList = () => {
```

Comment:
@niloofar-deriv Shouldn't we accept the `payload` as a prop? 🤔

Reply:
Done :pray: